bin/build

```diff
@@ -4,12 +4,15 @@
 # usage: ./bin/build
 set -ex
 
-. bin/build_utils
-
-CURRENT_DIR=$($(dirname $0)/abspath)
+CURRENT_DIR=$(cd $(dirname ${BASH_SOURCE[0]}); pwd)
 TOPLEVEL_DIR="$CURRENT_DIR/.."
 KEEP_ALIVE=${KEEP_ALIVE:-}
 
+# source build utilities
+pushd ${CURRENT_DIR}
+  . ./build_utils
+popd || true
```

Comment:
`popd` should never fail if you got into the directory properly. If you're trying to hide the output, use `&>/dev/null`